src/database/schema/ChatSchema.ts

```diff
@@ -211,7 +211,7 @@ class ChatSchema implements BaseDbSchema {
       channelId: json?.channel_id,
       userId: json?.message?.user?.id,
       message: json?.message?.text || json?.message?.message,
-      type: json?.message?.type,
+      type: json?.message?.message_type ?? json?.message?.type,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change here is adding an additional fallback for the `type` property. It now falls back to `json?.message?.message_type` before falling back to `json?.message?.type`. This is a good change if there are cases where the type could be under `message_type` instead of `type`.
```diff
-      type: json?.message?.type,
+      type: json?.message?.message_type ?? json?.message?.type,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->